cosmos/config.py

```diff
@@ -398,6 +398,8 @@ class ExecutionConfig:
     :param dbt_project_path: Configures the DBT project location accessible at runtime for dag execution. This is the project path in a docker container for ExecutionMode.DOCKER or ExecutionMode.KUBERNETES. Mutually Exclusive with ProjectConfig.dbt_project_path
     :param virtualenv_dir: Directory path to locate the (cached) virtual env that
     should be used for execution when execution mode is set to `ExecutionMode.VIRTUALENV`
+    :param async_py_requirements: In the case Execu....Creates a virtual environment with the specified dependencies. Example:
```

Comment:
Also, we have a parameter `py_requirements` similar to this, but it is part of the `operator_args` param, so I'm not sure if we should experiment here at least in docstring.

Reply:
yes, thanks. added experimental alongside `ExecutionMode.AIRFLOW_ASYNC`